File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/contabilita/RegistrazioneContabile.cs
using AgronicaCoreModelsSTD.anagrafiche;
using AgronicaCoreModelsSTD.baseClass;
using AgronicaCoreModelsSTD.metaschema.utilizzi;

namespace AgronicaCoreModelsSTD.contabilita
{
    public class RegistrazioneContabile : BaseCodeDescr
    {
        // movimento.doc_numero-->codice
        // movimento.mov_desc-->descrizione

        // movimento.doc_numero_sin
        public string rifDocumento { get; set; }

        // movimento.doc_numero_des
        public string anno { get; set; }

        // movimento.colli
        public int colli { get; set; }

        public RisorseUmane contraente { get; set; }

        public RegistrazioneContabile(int codice) : base(codice,"")
        {
        }

        public RegistrazioneContabile(int codice, string descrizione) : base(codice, descrizione)
        {
        }

        public RegistrazioneContabile() :base() { }

    }
}

File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/attivita/contabilita/RegistrazioneContabile.ts
import {BaseCodeDescr} from "../../baseClass/baseCodeDescr";
import {RisorseUmane} from "../../anagrafiche/RisorseUmane";

export class RegistrazioneContabile extends BaseCodeDescr {

    rifDocumento: string;

    anno: string;

    colli: number;

    contraente: RisorseUmane;

    constructor(codice: number, descrizione?: string) {
        super(codice,descrizione);
    }

}
